rethinkdb_base_block.py

```diff
@@ -1,18 +1,15 @@
 import rethinkdb as rdb
-
 from nio.block.base import Block
+from nio.block.mixins import Retry, LimitLock
+from nio.block.mixins import EnrichSignals, LimitLock
 from nio.command import command
```

Comment:
Command and rdb are both unused. Multiple import on the LimitLock. EnrichSignals not used